src/main/java/use_case/search/SearchOutputBoundary.java

```diff
@@ -0,0 +1,20 @@
+package use_case.search;
+
+/**
+ * Output boundary for the search by criteria use case.
+ */
+public interface SearchOutputBoundary {
+
+    /**
+     * Update the UI with the results of the search.
+     *
+     * @param outputData the search results to be displayed in the UI.
+     */
+    void displaySearchResults(SearchByCriteriaOutputData outputData);
```

Comment:
In accordance with my comment in the Presenter, I think it would follow our current naming conventions if this were called the "prepareSuccessView" method.

```suggestion
    void prepareSuccessView(SearchByCriteriaOutputData outputData);
```